charts/nri-bundle/requirements.lock

```diff
@@ -1,21 +1,24 @@
 dependencies:
 - name: newrelic-infrastructure
   repository: file://../newrelic-infrastructure
-  version: 1.6.4
+  version: 1.6.5
 - name: nri-prometheus
   repository: file://../nri-prometheus
-  version: 1.3.2
+  version: 1.4.0
 - name: nri-metadata-injection
   repository: file://../nri-metadata-injection
-  version: 1.2.0
+  version: 1.3.0
 - name: kube-state-metrics
   repository: https://charts.helm.sh/stable
   version: 2.9.1
 - name: nri-kube-events
   repository: file://../nri-kube-events
-  version: 1.3.4
+  version: 1.4.0
 - name: newrelic-logging
   repository: file://../newrelic-logging
   version: 1.4.2
-digest: sha256:e3debc8b192fe2ee26342de591855d4ff9e80f6253bb7710861ec2315e8e5d1a
-generated: "2020-11-26T12:46:14.62338+01:00"
+- name: nri-statsd
```

Comment:
Can we split the PR of the new chart from the update of nri-bundle?

Reply:
@paologallinaharbur 👍 definitely, that sounds good. I stripped out all of the new chart references from this PR. Once this is merged I can open another to update nri-bundle. Does that sound alright?